packages/overlay/src/active-overlay.ts

```diff
@@ -399,7 +399,6 @@ export class ActiveOverlay extends LitElement {
     }
 
     public renderTheme(content: TemplateResult): TemplateResult {
-        import('@spectrum-web-components/theme');
```

Comment:
Unless you've manually placed yourself of this path, there is no other way to get to this method without having included some part of this package in your application, so there's no reason to force ALL of it on you at this point.